packages/drupal-decoupled/src/editor/config.ts

```diff
@@ -0,0 +1,340 @@
+import { isComposable, isReference, isReferenceRequired } from "./helpers";
+import { Parser, Preset, WithPresetProps, WithOperationProps } from "./parser";
+import { Data, Field, FieldConfig, FieldConfigExtends, FieldExtend, FieldPreset } from "./types";
```

Comment:
```suggestion
import type { Data, Field, FieldConfig, FieldConfigExtends, FieldExtend, FieldPreset } from "./types";
```

Just to help ts compiler